pkg/health/health.go

```diff
@@ -32,6 +33,31 @@ var (
 	}
 )
 
+// CheckManagedResource checks if the conditions of ManagedResources are healthy.
+// It additionally checks whether at least the `v1alpha1.ResourcesApplied` has been reported.
+func CheckManagedResource(mr v1alpha1.ManagedResource) error {
```

Comment:
```suggestion
func CheckManagedResource(mr *v1alpha1.ManagedResource) error {
```

Reply:
Why do you want to pass a pointer? `CheckManagedResource` does not modify the object.

Reply:
To avoid a copy. All other check funcs accept pointers. However you can ignore the comment if you don't find it relevant.

Reply:
Okay you are right, for sake of consistency.